validation/pipeline/qase/schemaupload/main.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"github.com/rancher/tests/validation/pipeline/qase"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	_, callerFilePath, _, _ = runtime.Caller(0)
+	basepath                = filepath.Join(filepath.Dir(callerFilePath), "..", "..", "..", "..")
+)
+
+func main() {
+	qase.SetupQaseClient()
+
+	schemaMap, err := getRawSchemaData()
+	if err != nil {
+		logrus.Error("Error retrieving test schemas: ", err)
+		return
+	}
+
+	for project, schemas := range schemaMap {
+		var cases []qase.TestCase
+		for _, schema := range schemas {
+			suite := extractSubstrings(schema, `## Test Suite: (.+)\n`)[0][1]
```

Comment:
would be nice to document somewhere that `## Test Suite: ` is required in the schema

Reply:
i.e. what if there's an error / typo in schemas.md, will this create false / erroneous suites?

Reply:
yeah I was trying to have the documentation with the pit_schemas.md example file that's in there. The overall structure of the file is very important to follow, and if it's not followed then it will definitely lead to weird errors... I tried to do MD file unmarshalling but things got very messy and I didn't like it.